transaction-pool/src/status.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+/// Light pool status.
+/// This status is cheap to compute and can be called frequently.
+#[derive(Debug, PartialEq, Eq)]
+pub struct LightStatus {
+	/// Memory usage in bytes.
+	pub mem_usage: usize,
+	/// Total number of transactions in the pool.
+	pub count: usize,
```

Comment:
Better to `s/count/transaction_count/`. More status fields may be added in future which may cause confusion.